app/tests/test_app.py

```diff
@@ -820,3 +820,24 @@ def test_mobile_number_internationalisation(self):
 
         num = utils.mobile_number_to_international('0123456789')
         self.assertEqual(num, '27123456789')
+
+class AdminActionsTestCase(TestCase):
+    def setUp(self):
+        Site.objects.create(id=2, name='french', domain='fr.site.com')
+        client = Client()
```

Comment:
so yes :+1: this can be removed, it's not being used. I'm surprised flake8 didn't catch this.